nemo/lightning/run/plugins.py

```diff
@@ -314,7 +314,8 @@ class PerfEnvPlugin(run.Plugin):
     enable_layernorm_sm_margin: bool = True
     layernorm_sm_margin: int = 16
     enable_vboost: bool = False
-    nccl_pp_comm_chunksize: int = None
+    nccl_pp_comm_chunksize: Optional[int] = None
+    custom_cuda_device_max_connections: int = None
```

Comment:
Can we use `num_cuda_device_max_connections` instead of `custom_cuda_device_max_connections`?